src/Stripe/Services/Customers/StripeCustomerCreateOptions.cs

```diff
@@ -28,6 +28,9 @@ public class StripeCustomerCreateOptions
         [JsonProperty("plan")]
         public string PlanId { get; set; }
 
+        [JsonProperty("tax_percent")]
+        public decimal? TaxPercent { get; set; }
```

Comment:
Hi.
   I tried to use TaxPercent but I got an error: "stripe net Invalid decimal: 23.0000 must contain at maximum two decimal places."
I downloaded the source code, changed TaxPercent from decimal to string and problem solved since I was able to send formatted string.